docs/ashrae9012022/section6/Rule6-12.md

```diff
@@ -0,0 +1,43 @@
+
+# Lighting - Rule 6-12  
+
+**Rule ID:** 6-12   
+**Rule Description:** In buildings >5000 ft2 lighting shall be modeled having occupancy sensors in employee lunch and break rooms, conference/meeting rooms, and classrooms (not including shop classrooms, laboratory classrooms, and preschool through 12th grade classrooms). These controls shall be reflected in the baseline building design lighting schedules.  
+**Rule Assertion:** Baseline RMD = expected value  
+**Appendix G Section:** G3.1 6 Baseline
+
+**Mandatory Rule:** True  
+**Evaluation Context:** Each Space  
+**Function Call:**  
+
+- get_component_by_id()
+
+## Applicability Check:  
+1. Building area is greater than 5000 ft2
+2. Space type is one of the following: employee lunch and break rooms, conference/meeting rooms, and classrooms (not including shop classrooms, laboratory classrooms, and preschool through 12th grade classrooms)
+
+## Rule Logic:
+- For each building in the baseline model: `for building_b in B_RMD...buildings:`
+  
+  **Applicability Check 1:** 
+  - Declare a variable for the building area: `building_area_b = 0`
+  - Iterate the spaces in the building: `for space_b in building_b...spaces:`
+     - Add the area of each space to the building area sum: `building_area_b += space_b...floor_area`
+  - Check if the building area is greater than 5000 ft2: `if building_area_b > 5000: IS_APPLICABLE`
+  
+    - For each space in the building: `for space_b in building_b...spaces:`
+    
+    - **Applicability Check 2:**
+      - Check if the space type is one of the specified types: `if space_b["lighting_space_type"] in [LOUNGE_BREAKROOM_ALL_OTHERS, CONFERENCE_MEETING_MULTIPURPOSE_ROOM, CLASSROOM_LECTURE_HALL_TRAINING_ROOM_PENITENTIARY, CLASSROOM_LECTURE_HALL_TRAINING_ROOM_ALL_OTHER] or (space_b["function"] == LABORATORY and space_b["lighting_space_type"] == CLASSROOM_LECTURE_HALL_TRAINING_ROOM_SCHOOL): IS_APPLICABLE`
```

Comment:
For getting at Laboratory can you just use the space type since it is an actual space type in 90.1? 
<img width="396" height="100" alt="image" src="https://github.com/user-attachments/assets/02c7aa37-8377-4829-aa7b-eefc5b3dc0a3" />

Reply:
I don't see an issue with keeping that as well.